api/libproject/devstart.rb

```diff
@@ -456,6 +456,19 @@ def run_bigquery_tests(cmd_name, *args)
   :fn => ->(*args) { run_bigquery_tests("bigquerytest", *args) }
 })
 
+def run_rainforest_tests(cmd_name, *args)
+  ensure_docker cmd_name, args
+  common = Common.new
+  token = `gsutil cat gs://all-of-us-rw-staging-credentials/rainforest-key.txt`
```

Comment:
Is hardcoding to staging intentional? If so why? (please add a comment)

Reply:
I did want to hardcode to staging, because thats the environment that Rainforest runs against. We do have an identical key in each environment though.